docker/centos7/Dockerfile

```diff
@@ -23,11 +23,12 @@ RUN if [ ! "$(uname -m)" == "aarch64" ]; then \
         curl \
         debbuild \
         devtoolset-8 \
```

Comment:
This line looks, to me, like it should have been replaced by the line below. Is there a reason to keep this package?

Reply:
Done. I left it in during testing and accidentally forgot to remove it.